cargo-dist/templates/installer/npm/run.js.j2

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env node
 
 const { run } = require("./binary");
+{%- if bin is defined %}
```

Comment:
I don't mind us deferring this to later but we should add a comment so we understand why we did this.

Reply:
https://github.com/axodotdev/cargo-dist/issues/1524